addons/crm/models/res_partner.py

```diff
@@ -63,3 +63,15 @@ def action_view_opportunity(self):
         else:
             action['domain'] = [('partner_id.id', '=', self.id)]
         return action
+
+    def _fields_sync(self, values):
+        # sync salesteam with parent
+        if not values.get('team_id') and values.get('parent_id') and self.company_type == 'person':
```

Comment:
what happen if I do `partner.parent_id = 5` if a team is set, I think the team will be overridden

I think it should be
```suggestion
        if not self.team_id and not values.get('team_id') and values.get('parent_id') and self.company_type == 'person':
```

It should be good to add a test when you write on parent_id only but the team is set (to check that the team is not changed)